contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.squareup.contour.wrappers
+
+import android.graphics.Rect
+import com.google.common.truth.Truth.assertThat
+import com.squareup.contour.XInt
+import com.squareup.contour.YInt
+import com.squareup.contour.constraints.SizeConfig
+import org.junit.Test
+
+class ParentGeometryTest {
+
+  private val lambda: (Int) -> Int = { it }
+  private val paddingConfig = {
+    Rect().apply {
+      top = 10
+      right = 10
+      left = 10
+      bottom = 10
+    }
+  }
+  private val mockWidthConfig = SizeConfig(available = 40, lambda = lambda)
+  private val mockHeightConfig = SizeConfig(available = 40, lambda = lambda)
```

Comment:
No need for the `mock` prefix. And I would use a different value for height and width.

Reply:
Updated in 90036f8